### DISCUSSION TITLE
Up Threads to 96

### DISCUSSION THREAD

**Discussion by BenoitBe** [vdemichev/DiaNN#624] [Q&A]
Dear Vadim,

Is it possible to up threads more than 48?

If I use the window, don't seem to work.

![image](https://user-images.githubusercontent.com/90833041/220311516-da0b8d21-578d-4537-a452-1542bcd2e4d9.png)

I used instead the command line "--threads 96". Is it correct?

Actually my computer has  256 Go ram and 4x24 core (96 core).

Thanks in advance,

Benoit

**Comment by vdemichev** [comment]
Hi Benoit,

Yes, --threads shall set the desired number of threads.
But it's been reported that it does not work with large numbers beyond 32 or so.
Anyway, you can always run multiple DIA-NN instances in parallel.

Best,
Vadim
